src/main/java/com/iggroup/universityworkshopmw/domain/model/Market.java

```diff
@@ -0,0 +1,16 @@
+package com.iggroup.universityworkshopmw.domain.model;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+
+@ToString
```

Comment:
Maybe we could add @EqualsAndHashCode on the domain objects too (as it's normally included with @Data)